### DISCUSSION TITLE
Set a minimum and maximum absolute volume level for announcements

### DISCUSSION THREAD

**Discussion by rcjkierkels** [music-assistant/support#1018] [Feature requests and ideas]
**Request**
Make a slider so you can set the minimum and max absolute levels for announcements. If the current volume of the player is less than the minimum level, the announcement will be louder than the current music, because it will be at minimum level.

**Description**
Currently you can only set a relative volume level for announcements. This does not work in all cases, for example when we play some background music on our Google Audio and an announcement is being made you can barely hear it because it only increases the volume for say 20%. Just increasing this relative volume will also not always work because sometimes this is just too loud. I want my announcements always to be on say level 40% when the current volume level is less than 40%. So even if we play music on say 20% volume the announcement is still made on 40% volume level. But when we are having a party and the volume of the music is 60% the announcement should also be on 60% because there might be a lot of background noise. But there should be an option to limit this also.

**Comment by rjgrandy** [comment]
Agree with this!

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
